app/services/delete_remote_tags.rb

```diff
@@ -0,0 +1,10 @@
+class DeleteRemoteTags < RemoteTaggingService
```

Comment:
Same question about the class name vs input argument.

Reply:
@bzwei We are unlink one workflow at a time, when we unlink multiples we can can change the name

Reply:
@bzwei Missed it I will fix the inputs